Src/FluentAssertions/Specialized/AsyncFunctionAssertions.cs

```diff
@@ -366,4 +404,48 @@ private static async Task<Exception> InvokeWithInterceptionAsync(Func<Task> acti
             return exception;
         }
     }
+
+    private async Task<Exception> InvokeWithInterceptionAsync(TimeSpan timeSpan)
```

Comment:
🔧 This method should be placed directly under the calling `ThrowWithinAsync`
🔧 I think renaming `timeSpan` to `timeOut` makes a bit more self-explanatory

Reply:
I think `timeout` is the correct casing?

Reply:
Yeah, since we're following the [US spelling](https://grammarist.com/spelling/timeout-time-out/), I guess you're right

Reply:
Ok, done.
So far I supposed ordering by member type ("public", "private", "static", etc.).

Reply:
No, we order according to https://csharpcodingguidelines.com/layout-guidelines/#AV2406.